Gemfile.lock

```diff
@@ -576,5 +576,8 @@ DEPENDENCIES
   uglifier
   webmock
 
+RUBY VERSION
```

Comment:
I think the existence of this is a good thing https://devcenter.heroku.com/articles/ruby-versions#ruby-version-specifiers.